architecture/azure/messaging.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Azure messaging
+summary:
+reviewed: 2023-07-18
+callsToAction: ['solution-architect', 'poc-help']
+---
+
+Azure offers [several messaging services](https://learn.microsoft.com/en-us/azure/service-bus-messaging/compare-messaging-services), each built for specific purposes. The Particular Service Platform supports [Azure Service Bus and Azure Queue Storage](https://learn.microsoft.com/en-us/azure/service-bus-messaging/service-bus-azure-and-service-bus-queues-compared-contrasted).
+
+### Azure Service Bus
+
+[Azure Service Bus](https://learn.microsoft.com/en-us/azure/service-bus-messaging/service-bus-messaging-overview) is a fully-fledged enterprise messaging service with support for queuing, publish/subscribe, and more advanced integration patterns. It is designed to integrate applications or application components that may span multiple communication protocols, data contracts, trust domains, or network environments.
+
+Azure Service Bus has two pricing tiers: [Standard and Premium](https://learn.microsoft.com/en-us/azure/service-bus-messaging/service-bus-premium-messaging). The Premium tier provides hardware-level isolation, which means each customer's workload runs on dedicated hardware. Premium tier uses _messaging units_ for scaling and pricing and guarantees predictable throughput and performance. The Standard tier offers pay-as-you-go pricing with quota-based throughput and non-guaranteed latency. Microsoft recommends running production systems on the Premium tier.
+
+:heavy_plus_sign: Pros:
+
+- Up to 100 MB message size on the Premium tier
+- Supports transactional message operations across multiple queues and topics (cross-entity transactions)
+- Scripted infrastructure deployment using the [NServiceBus ASB CLI](/transports/azure-service-bus/operational-scripting.md)
+- Supports the AMQP 1.0 protocol over TCP and WebSockets
+
+:heavy_minus_sign: Cons:
+
+- Some features are only available on the Premium tier
+- The maximum message size is 256 KB on the Standard tier
+- The maximum queue size is 80 GB on the Standard tier
+- Cross-entity transactions are limited to 100 messages
+- No cross-namespace communication
+- No emulator for local development or testing available
+
+
+[**Try the Azure Service Bus Tutorial →**](https://learn.microsoft.com/en-us/azure/service-bus-messaging/build-message-driven-apps-nservicebus?tabs=Sender)
+
+#### When to use Azure Service Bus
+
+Azure Service Bus may be considered the default messaging choice for Azure. Alternatives should be considered only when the message size limitations are not sufficient or if a more portable queueing technology is required.
```

Comment:
I'd be tempted to go further than my minor edit above and remove the bit about message size restrictions and maybe the thing about portability too. IMO, the only reason to go ASQ is if you're cash-strapped.